dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -60,6 +60,47 @@ def __add__(self, other):
                                  other.__calculations)
         return merged_profile
 
+    def diff(self, other_profile, options=None):
+        """
+        Finds the differences for CategoricalColumns.
+
+        :param other_profile: profile to find the difference with
+        :type other_profile: CategoricalColumn
+        :return: the CategoricalColumn differences
+        :rtype: dict
+        """
+        differences = super().diff(other_profile, options)
+
+        differences["categorical"] = \
+            utils.find_diff_of_strings_and_bools(self.is_match, other_profile.is_match)
+
+        differences["statistics"] = dict([
+                ('unique_count', utils.find_diff_of_numbers(len(self.categories),
+                                                            len(other_profile.categories))),
+                ('unique_ratio', utils.find_diff_of_numbers(self.unique_ratio, 
+                                                            other_profile.unique_ratio)),
+            ])
+
+        # These stats are only diffed if both profiles are categorical
+        if self.is_match and other_profile.is_match:
+            differences["statistics"]['categories'] = \
+                utils.find_diff_of_lists_and_sets(self.categories,
+                                                  other_profile.categories)
+            differences["statistics"]['gini_impurity'] = \
+                utils.find_diff_of_numbers(self.gini_impurity,
+                                           other_profile.gini_impurity)
+            cat_count1 = dict(sorted(self._categories.items(),
+                                     key=itemgetter(1),
+                                     reverse=True))
+            cat_count2 = dict(sorted(other_profile._categories.items(),
+                                     key=itemgetter(1),
+                                     reverse=True))
+
+            differences["statistics"]['categorical_count'] = \
+                utils.find_diff_of_dicts(cat_count1, cat_count2)
```

Comment:
Just noting this here, depending on which PR goes in first, will need to have someone add unalikability here once #320 is in.

Reply:
Yea, lets get alberts in first